Makefile

```diff
@@ -254,3 +254,13 @@ start-shoot-cert-service:
 		--ignore-operation-annotation=$(IGNORE_OPERATION_ANNOTATION) \
 		--leader-election=$(LEADER_ELECTION) \
 		--config=./controllers/extension-shoot-cert-service/example/00-config.yaml
+
+.PHONY: start-gardener-validator-aws
+start-gardener-validator-aws:
+	@LEADER_ELECTION_NAMESPACE=garden GO111MODULE=on go run \
+		-mod=vendor \
+		-ldflags $(LD_FLAGS) \
+		./controllers/provider-aws/cmd/gardener-validator-aws \
+		--webhook-config-server-host=0.0.0.0 \
+		--webhook-config-server-port=8443 \
+		--webhook-config-cert-dir=$(CERT_DIR)
```

Comment:
where is CERT_DIR defined in this case?

Reply:
At the moment you have to defined yourself.

I can also create a dummy PKI with a certificate for `localhost` and add it to the example folders. Then I can provide a decent default for `CERT_DIR`. WDYT?

Reply:
Even better :)